Form-Controls/styles.css

```diff
@@ -0,0 +1,88 @@
+main {
+  display: flex;
+
+  justify-content: center;
+  align-items: center;
+}
+#form-info {
+  padding: 20px;
+  background-color: rgba(199, 197, 203, 0.44);
+  width: 500px;
+  height: auto;
+  border: 1px solid rgb(172, 170, 170);
+}
+
+#btn {
+  padding: 0.25em 0.75em;
+  min-width: 10ch;
+  min-height: 30px;
+  text-align: center;
+  line-height: 1.1;
+  background-color: rgb(163, 139, 159);
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
+  font-weight: bold;
+  border-radius: 45%;
+  align-items: center;
+}
+
+#btn:hover {
+  text-shadow: 2px;
+  color: rgb(8, 19, 221);
+}
+.form-control {
+  font-size: 16px;
+  font-size: max(16px, 1em);
```

Comment:
can you talk through this rule? What are you trying to achieve here?

Reply:
I removed it ** font-size: max(16px, 1em);** , when I was checking for lighthouse Accessibility I forgot to remove it.